pom.xml

```diff
@@ -124,6 +122,36 @@
     </dependency>
   </dependencies>
 
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.jacoco</groupId>
+        <artifactId>jacoco-maven-plugin</artifactId>
+        <version>0.8.10</version>
+        <executions>
+          <execution>
+            <goals>
+              <goal>prepare-agent</goal>
+            </goals>
+          </execution>
+          <execution>
+            <id>report</id>
+            <phase>test</phase>
+            <goals>
+              <goal>report</goal>
+            </goals>
+          </execution>
+        </executions>
+        <configuration>
+          <excludes>
+            <exclude>com/synopsys/integration/jenkins/scan/extension/**/*.class</exclude>
+            <exclude>com/synopsys/integration/jenkins/scan/SecurityScanner.class</exclude>
```

Comment:
@kishorikumar @dmamidisynopsys and @devmehtasynopsys 
Intentionally excluded these **Jenkins API** specific classes from the code coverage.
 Let me know your thoughts!

Reply:
Please test this locally and through pipeline before merging. Looks good overall

Reply:
I am testing on pipeline and will edit if necessary

Reply:
@jahid1209 @kishorikumar @devmehtasynopsys Lets not merge this change yet, I see new requirements (few additional) from Jenkins Infra team